jest.config.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2019 Google LLC
```

Comment:
AMP Project OSS Boilerplate is slightly different.

https://github.com/ampproject/worker-dom/blob/master/src/transfer/Messages.ts#L2

Reply:
Done.